### DISCUSSION TITLE
throws out of any vehicle from GTA Online

### DISCUSSION THREAD

**Discussion by Vonilaz** [MAFINS/MenyooSP#673] [Q&A]
The problem, I suppose, is quite ususual.
When I get into any vehicle from GTA Online, I will be teleported nearby immediately or for a while. I tried restarting GTA V without internet and restarting the computer.
Video - https://file.io/YRcDRwVJRjUj
             https://fastupload.io/5e5c1f2bffade0ec

**Comment by metoxys** [comment]
Duplicate of #579
